notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<p>please add a step to save the output to <code>data/processed/metrics</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
:+1: